src/ex2/PokemonClient.js

```diff
@@ -0,0 +1,30 @@
+class PokemonClinet {
+  constructor() {
+
+  }
+  fetchPokemon(arr) {
```

Comment:
use meaningful names
```suggestion
  fetchPokemon(pokemonIds) {
```